typings/index.d.ts

```diff
@@ -11,10 +11,11 @@ declare module "discord.js" {
     export const version: string;
     export class Client extends EventEmitter {
         constructor(options?: ClientOptions);
+        browser: boolean;
+        channels: Collection<string, Channel>;
         email: string;
         emojis: Collection<string, Emoji>;
         guilds: Collection<string, Guild>;
-        channels: Collection<string, Channel>;
         options: ClientOptions;
         password: string;
```

Comment:
Might wanna remove email and password here since it already does not work anymore on indev.